javascript/README.md

```diff
@@ -1,7 +0,0 @@
-# Javascript Sample Projects
-
-These sample projects leverage the [mapepire-js](https://github.com/Mapepire-IBMi/mapepire-js) client SDK:
-
-* `<project-1>`: `<project-1-description>`
```

Comment:
Should we move this project into a subdirectory under the typescript folder so other projects can be added later and then add this file back?

Reply:
Yes. But, let's do that in a new PR.